Jenkinsfile

```diff
@@ -307,29 +305,3 @@ def stopDocker() {
     sh("make stop")
   }
 }
-
-def alertTestOutcome(params, testStatus) {
-  def channel = "#govuk-e2e-tests"
-  // post to slack just when it's an important branch
-  if (env.BRANCH_NAME == "main" && (testStatus.mainFailed || testStatus.startUpFailed)) {
-    def message = "Publishing end-to-end tests <${BUILD_URL}|failed> for main branch, changes not pushed to test-against"
-    slackSend(color: "#d40100", channel: channel, message: message)
-  } else if (env.BRANCH_NAME == "test-against" && testStatus.startUpFailed) {
-    def message = "Publishing end-to-end tests start up <${BUILD_URL}|failed> for $NODE_NAME"
-    slackSend(color: "#d40100", channel: channel, message: message)
-  } else if (env.BRANCH_NAME == "test-against" && testStatus.mainFailed) {
-    def message = "Publishing end-to-end tests <${BUILD_URL}|failed>"
-    message += (params.ORIGIN_REPO) ? " for ${params.ORIGIN_REPO}" : ""
-    slackSend(color: "#d40100", channel: channel, message: message)
-  } else if (env.BRANCH_NAME == "test-against" && testStatus.flakyNewFailed) {
-    def message = "Publishing end-to-end flaky/new tests <${BUILD_URL}|failed>"
-    message += (params.ORIGIN_REPO) ? " for ${params.ORIGIN_REPO}" : ""
-    slackSend(color: "#ffff94", channel: channel, message: message)
-  }
-
-  if (testStatus.mainFailed) {
-    def guideUrl = "https://github.com/alphagov/publishing-e2e-tests/blob/main/CONTRIBUTING.md#dealing-with-flaky-tests"
-    currentBuild.description = "<p style=\"color: red\">Is the failure unrelated to your change?</p>" +
-                               "<p>We have <a href=\"${guideUrl}\">flaky test advice available</a> to help.</p>"
-  }
```

Comment:
This isn't part of the slack sending - it's part of the Jenkins information. Though I think it can be removed as it now links to a 404

Reply:
Oh yes, thanks for spotting. Done: https://github.com/alphagov/publishing-e2e-tests/pull/475/commits/6c3dbfbd02b6f2bfb7eb4ee1133b6d0659a1ac63